app/src/main/res/values/settings_keys.xml

```diff
@@ -24,6 +24,8 @@
     <string name="screen_brightness_timestamp_key">screen_brightness_timestamp_key</string>
     <string name="clear_queue_confirmation_key">clear_queue_confirmation_key</string>
     <string name="ignore_hardware_media_buttons_key">ignore_hardware_media_buttons_key</string>
+    <string name="share_playlist_with_details_key">share_playlist_with_details_key</string>
+    <string name="share_playlist_with_details_can_show_dialog_key">share_playlist_with_details_can_show_dialog_key</string>
```

Comment:
I think this name is better to understand what the key is used for. I think the usage needs to be inverted though.
```suggestion
    <string name="remember_local_playlist_sharing_option">remember_local_playlist_sharing_option</string>
```